src/debugAdapter/goDebug.ts

```diff
@@ -511,25 +521,25 @@ class GoDebugSession extends DebugSession {
 					verbose('Error on CreateBreakpoint');
 					return null;
 				});
-			}));
-		}).then(newBreakpoints => {
-			verbose('All set:' + JSON.stringify(newBreakpoints));
-			let breakpoints = newBreakpoints.map((bp, i) => {
-				if (bp) {
-					return { verified: true, line: bp.line };
-				} else {
-					return { verified: false, line: args.lines[i] };
-				}
+			})).then(newBreakpoints => {
```

Comment:
It looks like you moved this code up a level in the promise chain (see the indentation). Was that necessary? If not, can you leave it the same?